index.html

```diff
@@ -4822,15 +4822,17 @@ <h2>Level of Assurance</h2>
     <h1>Privacy Considerations</h1>
 
     <p>
-It is critically important to apply the principles of Privacy by Design
-[[PRIVACY-BY-DESIGN]] to all aspects of the <a>decentralized identifier</a>
-architecture, because <a>DIDs</a> and <a>DID documents</a> are, by design,
-administered directly by the <a>DID controller(s)</a>. There is no registrar,
-hosting company, or other intermediate service provider to recommend or apply
-additional privacy safeguards. The authors of this specification have applied
-all seven Privacy by Design principles throughout its development.
-Privacy in this specification is preventative not remedial, and privacy is an
-embedded default.
+Since <a>DIDs</a> and <a>DID documents</a> are designed to be administered
+directly by the <a>DID controller(s)</a>, it is critically important to apply
+the principles of Privacy by Design [[PRIVACY-BY-DESIGN]] to all aspects of the
+<a>decentralized identifier</a> architecture. All seven Privacy by Design
+principles have been applied throughout the development of this specification.
+The design used in this specification does not assume that there is  a
+registrar, hosting company, or other intermediate service provider to recommend
+or apply additional privacy safeguards. Privacy in this specification is
+preventative, not remedial, and is an embedded default. The following sections
+cover privacy considerations that implementers might find useful when
+building systems that utilize <a>decentralized identifiers</a>.
```

Comment:
```suggestion
<a>decentralized identifier</a> architecture. All seven of these principles
have been applied throughout the development of this specification. The design
used in this specification does not assume that there is a registrar, hosting
company, nor other intermediate service provider to recommend or apply
additional privacy safeguards. Privacy in this specification is preventive,
not remedial, and is an embedded default. The following sections cover privacy
considerations that implementers might find useful when building systems that
utilize <a>decentralized identifiers</a>.
```
I made a few word changes, which changed the wrap points... Hopefully the wording changes are clear enough.